lib/node_modules/@stdlib/blas/base/idamax/manifest.json

```diff
@@ -0,0 +1,171 @@
+{
+  "options": {
+    "os": "linux",
+    "blas": "",
+    "wasm": false
+  },
+  "fields": [
+    {
+      "field": "src",
+      "resolve": true,
+      "relative": true
+    },
+    {
+      "field": "include",
+      "resolve": true,
+      "relative": true
+    },
+    {
+      "field": "libraries",
+      "resolve": false,
+      "relative": false
+    },
+    {
+      "field": "libpath",
+      "resolve": true,
+      "relative": false
+    }
+  ],
+  "confs": [
+    {
+      "os": "linux",
+      "blas": "",
+      "wasm": false,
+      "src": [
+        "./src/idamax.f",
+        "./src/idamaxsub.f",
+        "./src/idamax_f.c"
+      ],
+      "include": [
+        "./include"
+      ],
+      "libraries": [],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/napi/export",
+        "@stdlib/napi/argv",
+        "@stdlib/napi/argv-int64",
+        "@stdlib/napi/argv-strided-float64array"
```

Comment:
why do we not have `abs` registered in dependency?

Reply:
Because the abs that is used in idamax.c is inbuilt C from math.h header and not from stdlib so dependency is not required here.